src/App.js

```diff
@@ -3,6 +3,7 @@ import { charityAPI } from './clients';
 import Logo from './components/Logo';
 import MainContact from './components/MainContact';
 import FeaturedBanner from './components/FeaturedBanner';
+import Sponsers from './components/sponsers';
```

Comment:
As a convention, We name React component files (or the directory that contains the file) capitalized. In your case, `components/Sponser`